tests/test_routes.py

```diff
@@ -178,6 +178,29 @@ def test_delete_order(self):
         )
         self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
 
+    def test_action_status(self):
+        """ change the status of an order to cancelled """
+        # create an Action to change order status to canceled
+        order_item = OrderItem(product_id=1, quantity=1, price=5, order_id = 1)
+        order = Order(customer_id=1, tracking_id=1, status=OrderStatus.Created, order_items=[order_item])
+        resp = self.app.post(
+            "/order", 
+            json=order.serialize(), 
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Action to cancel the order
+        #new_order = resp.get_json()
+        #new_order["status"] = OrderStatus.Cancelled
+        resp = self.app.put(
+            "/order/1/cancel"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        updated_order = resp.get_json()
```

Comment:
We should not check the response json here, we should fetch the order from the database again and check the status on that.

Reply:
oh alright, will do that!